README_CONTAINER_IMAGE.md

```diff
@@ -38,4 +44,20 @@ Here is an example of how to run a containerized `openshift-ansible` playbook th
            -e PLAYBOOK_FILE=playbooks/certificate_expiry/default.yaml \
            openshift/openshift-ansible
 
-The [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples) provide additional information on how to use an image built from it like this one.
+#### Scheduled job
```

Comment:
Is it possible to add some cross referencing to the documentation?

The [cert expiry role README](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_certificate_expiry/README.md) would be improved by adding some sort of section describing what you've done here and linking to your full documentation on the scheduled jobs.